client.go

```diff
@@ -298,7 +298,17 @@ func (c *Client) CancelTasks(param *TasksQuery) (resp *TaskInfo, err error) {
 		functionName:        "CancelTasks",
 	}
 	if param != nil {
-		encodeTasksQuery(param, &req)
+		paramToSend := &TasksQuery{
+			UIDS:             param.UIDS,
+			IndexUIDS:        param.IndexUIDS,
+			Statuses:         param.Statuses,
+			Types:            param.Types,
+			BeforeEnqueuedAt: param.BeforeEnqueuedAt,
+			AfterEnqueuedAt:  param.AfterEnqueuedAt,
+			BeforeStartedAt:  param.BeforeStartedAt,
+			AfterStartedAt:   param.AfterStartedAt,
+		}
+		encodeTasksQuery(paramToSend, &req)
```

Comment:
Can you make this function `encodeTasksQuery` support multiple sets of params?

Reply:
Not really, I could put `interface{}` as a parameter instead of a typed struct, but I wouldn't have access to the fields easily.